payments/swish/other-features.md

```diff
@@ -147,26 +147,17 @@ Content-Type: application/json
         }
     },
     "operations": [
-        {
-            "method": "PATCH",
-            "href": "http://{{ page.apiHost }}/psp/swish/payments/{{ page.paymentId }}",
-            "rel": "update-payment-abort"
-        },
         {
             "method": "POST",
             "href": "http://{{ page.apiHost }}/psp/swish/payments/{{ page.paymentId }}/sales",
             "rel": "create-sale"
         }
-
-    ]
 }
 ```
 
 {% include settlement-reconciliation.md %}
 
-{% include one-click-payments.md %}
-
-{% include payment-link.md %}
+{% include payment-link.md hide-3d-secure=true hide-authorization=true %}
```

Comment:
I see we have a `hide-3d-secure` parameter here. It should also be changed to `show-3d-secure` and have its logic negated.

```suggestion
{% include payment-link.md hide-3d-secure=true show-authorization=false %}
```